app/models/config.rb

```diff
@@ -43,6 +43,30 @@ class Config < ApplicationRecord
     :voicemail,
   ]
 
+  DEFAULTS = {
+    insurance: nil,
+    external_pledge_source: nil,
+    pledge_limit_help_text: nil,
+    language: nil,
+    resources_url: nil,
+    practical_support_guidance_url: nil,
+    fax_service: nil,
```

Comment:
ruby's hash accessor will return nil if it doesn't find a key, so if we wanted to take out the nils here we could. I'm fine either way